bobtemplates/plone_addon/.isort.cfg

```diff
@@ -0,0 +1,6 @@
+[settings]
+force_alphabetical_sort = True
+force_single_line = True
+lines_after_imports = 2
+line_length = 200
+not_skip = __init__.py
```

Comment:
I would rather put these settings on ``setup.cfg`` (then the header needs to be changed to ``[isort]`` so we keep less files around and we have one single place to put all the configurations.